skills/lark-drive/references/lark-drive-workflow-knowledgeOrganize.md

```
@@ -0,0 +1,624 @@
+权限问题分两类处理：
+
+1. 缺少 API scope：按 CLI hint 重新执行 `lark-cli auth login --scope ...` 或补齐对应 `--domain`。
+2. 当前用户没有目标资源访问权限：只能在用户明确确认后，使用 `drive +apply-permission` 发起单个资源的权限申请。该命令仅支持 `--as user`，会通知 owner，不允许批量自动调用。
+
+```bash
+lark-cli drive +apply-permission --token "<url_or_token>" --type <doc|docx|sheet|bitable|file|wiki|mindnote|slides> --perm view --as user
+```
+
+## 工作流
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)